etc/cvmfs/domain.d/computecanada.ca.conf

```diff
@@ -0,0 +1,10 @@
+# These are here so cvmfs will notice them if common.conf sets them
+CVMFS_HTTP_PROXY="$CVMFS_HTTP_PROXY"
+CVMFS_FALLBACK_PROXY="$CVMFS_FALLBACK_PROXY"
+. ../common.conf
+
+# See CVMFS_SERVER_URL in ../config.d/soft.computecanada.ca.conf
```

Comment:
Why define domain.d at all if it only works on /cvmfs/soft.computecanada.ca?

Reply:
For consistency, to include the common.conf file and set other common config like the keys dir.